internal/server/statvar_group.go

```diff
@@ -312,3 +343,36 @@ func computeSpecializedEntity(parentSvg string, childSvg string) string {
 	}
 	return strings.Join(result, ", ")
 }
+
+// Check if places have data for stat vars and stat var groups
+// Returns a map from stat var to the number of places that has data.
+func checkStatExistence(
+	ctx context.Context,
+	store *store.Store,
+	svOrSvgs []string,
+	places []string) (map[string]int, error) {
+	rowList, keyTokens := buildStatExistenceKey(places, svOrSvgs)
+	keyToTokenFn := tokenFn(keyTokens)
+	baseDataMap, _, err := bigTableReadRowsParallel(
+		ctx, store, rowList, func(string, []byte) (interface{}, error) {
+			// If exist, BT read returns an empty struct. Here just return nil to
+			// indicate the existence of the key.
+			return nil, nil
+		}, keyToTokenFn,
+	)
+	if err != nil {
+		return nil, err
+	}
+	result := map[string]int{}
+	for _, rowKey := range rowList {
+		placeSv := keyTokens[rowKey]
+		token, _ := keyToTokenFn(rowKey)
+		if _, ok := baseDataMap[token]; ok {
+			result[placeSv.statVar]++
+		} else {
+			// TODO(shifucun): remove the else part once cache is build.
+			result[placeSv.statVar]++
```

Comment:
This effectively making the filter an no-op. Will remove this after cache is build